pandas/io/tests/test_packers.py

```diff
@@ -363,7 +387,9 @@ def setUp(self):
             'D': date_range('1/1/2009', periods=5),
             'E': [0., 1, Timestamp('20100101'), 'foo', 2.],
             'F': [Timestamp('20130102', tz='US/Eastern')] * 5,
-            'G': [Timestamp('20130603', tz='CET')] * 5
+            'G': [Timestamp('20130603', tz='CET')] * 5,
+            'H': Categorical(['a', 'b', 'c', 'd', 'e']),
+            'I': Categorical(['a', 'b', 'c', 'd', 'e'], ordered=True),
```

Comment:
that check is not used at all! I was just going to tell you to take it out. a ``Categorical`` is fully serialized/deserialized via ``encode/decode`` the ``dtype`` is NEVER ``category``. except when its a ``series`` but that is already handled.